posts/test_post_uploading.py

```diff
@@ -69,12 +74,18 @@ def assert_posts_are_equal(self, first, second):
         self.assertEqual(first["files"], second["files"])
 
     def setUp(self):
-        """Set up mock DB for testing"""
+        """Set up mocks for testing."""
+        # mock db
         self.mock_collection = mongomock.MongoClient().db.collection
+        # mock Google storage module for uploading to storage bucket
+        app.storage = mock.MagicMock(spec=app.storage)
```

Comment:
Prefer using the [`@mock.patch('app.storage')` decorator format](https://docs.python.org/3/library/unittest.mock.html#quick-guide), which won't permanently overwrite `app.storage`..

Reply:
I used the path.start() and patch.stop() pattern instead to run the patch over the course of each step. https://github.com/knative-portability/large-events/pull/20/commits/5615d00b24da42270add718a294db18cfb8d9170